applications/post-editing.md

```diff
@@ -0,0 +1,9 @@
+---
+description: Linguistic editing of machine translation output
+---
+
+# Post-editing
+
+Post-editing is adapting the machine translation output to take it to human quality by fixing semantic errors, rearranging style, and adding or deleting information by comparing the output to source.
```

Comment:
Let's break it down more, and cover more motivation and less linguistic detail.

```
Human **post-editing** is the review and correction of machine translation output.

Usually the human translator can see both the output and the input.

It often replaces... because it is cheaper and faster.

Over the 2010s, with rising machine translation quality, it became the dominant human translation workflow.

### Flavors
With **full post editing**, the goal is to match human quality, as if human-translating from scratch.

With **light post-editing** ...

### Metrics
**Post-editing distance** ...

**Post-editing effort** ...

```